inline/tests/spec/snapshots/parser/verbatim/simple-verbatim.snap

```diff
@@ -0,0 +1,15 @@
+---
+source: inline/tests/parser/mod.rs
+info: "Test 'simple-verbatim' from 'verbatim.yml'"
+---
+Verbatim @ (1:1)->(1:11) (
+    ␢Verbatim␢
```

Comment:
why are there ␢?
If it is intentional, I think we should search for another symbol and use ␢ only to represent a backslash.
Otherwise it might be confusing to read.

Reply:
This is indeed intentional. The goal is to use `␢` as a way to show the mismatch between `Span` some element (e.g. `Inline`) occupies in input text, and the length of the actual content of that element. This way we can easily see that the calculated `Span` is indeed correct, even though the output is *"shorter"* than what the `Span` suggests.

Reply:
fine with me 👍 
(just a bit unfortunate that `␢` != escaped using **b**ackslash `\` 😄)